frontend/src/components/OtherEvaluation/introOtherEvaluation.js

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect, useState } from "react";
+import { Select } from "@code4ro/taskforce-fe-components";
+import PropTypes from "prop-types";
+import ProfileApi from "../../api/profileApi";
+import StartFormButton from "../Evaluation/startFormButton";
+
+const IntroOtherEvaluation = ({ onFinish }) => {
+  useEffect(() => {
+    ProfileApi.getDependants().then(members => {
+      setDependants(members);
+      setSelectedMember(members[0].id);
+    });
+  }, []);
+
+  const [dependants, setDependants] = useState(null);
+  const [selectedMember, setSelectedMember] = useState(undefined);
+
+  const props = {
+    onChange: function(el) {
+      setSelectedMember(el.target.value);
+    }
+  };
+
+  if (dependants === null) {
+    return <div>Formularul se incarca</div>;
+  }
+
+  if (dependants.length === 0) {
+    return <div>Nu aveti dependenti pentru care sa completati formulare</div>;
+  }
+
+  const options = dependants.map(dependant => {
+    return {
+      text: `${dependant.name} ${dependant.surname}`,
+      value: dependant.id,
+      selected: dependant.id === selectedMember
```

Comment:
Is this from our taskforce component? because using selected on options is not recommended (also throws an error in the console because of this)

Reply:
you're right! I looked at storybook and shows `selected`. Will check the code to see how to do it